@AH-64D Apache Longbow/Addons/fza_ah64_ase/functions/sensors/fn_lwr.sqf

```diff
@@ -0,0 +1,23 @@
+/* ----------------------------------------------------------------------------
+Function: fza_ase_fnc_lwr
+
+
+Description:
+    Proccessor function for the LASER WARNING RECIEVER
+
+Parameters:
+    heli: Object - Vehicle the event handler is assigned to
+
+Returns:
+    Nothing
+
+Examples:
+    [_heli] call fza_ase_fnc_lwr
+
+Author:
+    Snow(Dryden)
+---------------------------------------------------------------------------- */
+#include "\fza_ah64_ase\headers\ase.h"
+params ["_heli"];
+
+[]
```

Comment:
I see you have stub functions here for `lwr` and `cmws` - just leave them out until they are ready.